src/Network/Network.Test/ScenarioTests/AzureFirewallTests.ps1

```diff
@@ -1078,3 +1078,64 @@ function Test-AzureFirewallVirtualHubCRUD {
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Tests AzureFirewall AdditionalProperty
+#>
+function Assert-AreEqualHashtables {
```

Comment:
Please move this function to `Common.ps1`

Reply:
done!